massa-versioning/src/versioning.rs

```diff
@@ -908,11 +908,32 @@ impl MipStoreRaw {
                 {
                     let entry_value = e.get_mut();
                     *entry_value = entry_value.saturating_sub(1);
+                    network_version_count = *entry_value;
                     if *entry_value == 0 {
                         self.stats.network_version_counters.remove(&removed_version);
                     }
                 }
             }
+
+            if announced_network_version != 0 {
```

Comment:
didn't we remove the "0" special case ?

Reply:
We did in block header but we use it in mip store stats, when we received None we process it like it is nerwork version 0, this simplifies the code

Reply:
what if someone announces Some(0) ?

Reply:
Some(0) should be semantically the same as announcing None, because we can never roll back versions. (We can always introduce a new version that rolls back previous changes, but we can't go from active version of 2 to 0.